### ISSUE TITLE
http://juniper.graphql.rs/tutorial.html - 404

### ISSUE THREAD

**Issue by rofrol** [graphql-rust/graphql-rust.github.io#2]
Url is referenced in `source/docs/quickstart.md`.

**Comment by qguv** [comment]
Yes, looks dead to me too. Hard to develop for juniper when the site is down! :(

**Comment by mhallin** [comment]
GitHub had for some reason lost the custom domain name for this repository. This should be fixed now. As for the original issue with tutorial.html, I'll look into what happened there.

**Comment by qguv** [comment]
Unfortunately [the book](http://juniper.graphql.rs/juniper-book) is still 404ing for me.

**Comment by rofrol** [comment]
@qguv  You can use this address http://juniper.graphql.rs/

**Comment by rofrol** [comment]
@qguv

http://juniper.graphql.rs/ works for me.

How is http://juniper.graphql.rs/juniper-book different from http://juniper.graphql.rs/?

**Comment by mhallin** [comment]
http://juniper.graphql.rs/juniper-book does not exist. Has it been linked to from anywhere?

**Comment by qguv** [comment]
@mhallin It's indirectly linked by the description on this repository: https://github.com/graphql-rust/graphql-rust.github.io

**Comment by mhallin** [comment]
I see, that's a problem that's fixed now - it must have predated the move to the new domain. Thank you for reporting these issues.

**Comment by qguv** [comment]
Thanks @mhallin for fixing so quickly!

**Comment by rofrol** [comment]
This issue is about tutorial.md

**Comment by mhallin** [comment]
Yes, I realized that. Sorry for accidentally closing it.

**Comment by apopiak** [comment]
https://graphql-rust.github.io/tutorial.html is `404`ing as well. Referenced [here](https://graphql-rust.github.io/quickstart.html)

**Comment by Asoftyn** [comment]
Any updates on this? It`s still `404`ing
